requirements.txt

```diff
@@ -2,3 +2,5 @@ klein==0.2.3
 Twisted==15.5.0
 Werkzeug==0.10.4
 zope.interface==4.1.2
+python-dateutil==2.4.2
+sortedcontainers==1.4.4
```

Comment:
:grey_question: Prior to this PR, the deps are sorted, perhaps we should sort these so that is maintained?